content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "my-array-1"
+    static_configs:
+      - targets: ["my-array-1:8080"]
+```
+
+Create the Prometheus container:
+
+```markdown
+docker run -d --rm --name prometheus-server \
+    --network prometheus \
+    -p 80:9090 \
+    -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus:v2.30.3
+```
+
+In a few moments, the Prometheus server should be up and listening on the docker host and accessible from http://your.docker.host.example.com/. 
+
+To inspect if metrics are being collected properly, access the web interface and start typing "hpe" in the query bar.
+
+![Prometheus Query](/img/prometheus-auto.png)
+
+Select a \`volume_reads_per_second\` metric, hit "Execute" and click "Graph". This should produce a filled line graph with all the volumes on the array and their read IOPS. Wait a few minutes if there's no data being presented.
+
+![Prometheus Graph](/img/prometheus-graph.png)
+
+Next, we'll bolt on Grafana and connect to the Prometheus data source.
+
+# Run Grafana and Build Your First Dashboard
+
+In this tutorial we'll just use the built-in dashboards and plugins that come with Grafana. Accessing Grafana is mainly done through the intuitive web UI found on port 3000. Let's start it up:
+
+```markdown
+docker run --rm -d --name grafana \
+    --network prometheus \
+    -p 3000:3000 \
+    grafana/grafana:8.2.2
+```
+
+Next, navigate to the web UI. http://your.docker.host.example.com:3000/, login with user "admin" and password "admin", you'll be prompted to create a new password. Reusing "admin" for this example is perfectly fine.
+
+Next, a data source needs to be added. A panel to do so is available on the main screen.
+
+![Add new data source](/img/grafana-welcome.png)
+
+Next, select "Prometheus".
+
+![Select Prometheus](/img/grafana-select.png)
+
+In the HTTP section of the configuration, enter `http://prometheus-server:9090`. Click "Save & test" at the bottom of the page. A green checkmark should indicate that the "Data source is working".
+
+![Configure Prometheus data source](/img/grafana-server.png)
+
+At this point, Grafana is able to retrieve data series from Prometheus and dashboards may now be created.
+
+Hit the big "+" sign on the right, click "Dashboards" and "Add an empty panel".
+
+Creating a graph involves adding a query from the data source. The field uses auto-complete so start typing "hpe" and the field will populate. In the example below, a simple "sum" operation is performed on the `hpenimble_volume_reads_per_second_avg5m` metric which produces a combined read IOPS performance graph. The axis itself has been customized to show "io/s".
+
+![A simple Grafana dashboard](/img/grafana-panel.png)
+
+This concludes this introductory tutorial. Next, the possibilities are endless. Please find additional resources below on how learn more about Prometheus, Grafana and the HPE Storage Array Exporter for Prometheus.
```

Comment:
Recommend changing "Next, the possibilities are endless." to something a little more explanatory - something like "The possibilities for using this are endless, including use cases such as..."

(at the very least, I'd change it to "The possibilities for using this are endless."

Reply:
Added some verbiage.